src/Shared/FrameworkLocationHelper.cs

```diff
@@ -11,6 +11,7 @@
 using Microsoft.Win32;
 #endif
 
+using Microsoft.Build.Framework;
```

Comment:
this file should also be moved, check again which shared files are not included by MSBuildTaskHost. and all of those should be moved